R/datasets.R

```diff
@@ -1,3 +1,25 @@
+#' 
```

Comment:
Please move the documentation to the end of the file, not the beginning.

Reply:
@kaceyjj